rocketpool-daemon/api/pdao/status.go

```diff
@@ -125,13 +122,14 @@ func (c *protocolDaoGetStatusContext) PrepareData(data *api.ProtocolDaoStatusRes
 	var err error
 
 	data.IsVotingInitialized = c.node.IsVotingInitialized.Get()
-	if data.IsVotingInitialized {
+
+	if !data.IsVotingInitialized {
+		data.TotalDelegatedVp = nil
```

Comment:
if `data` is 0-valued when this function is called (and i believe it is) this is a nop and shouldn't be done